org.gecko.vaadin.generator/bnd.bnd

```diff
@@ -1,5 +1,7 @@
 -buildpath: \
 	osgi.annotation;version='7.0.0',\
+	osgi.core;version='7.0.0',\
```

Comment:
The generator is not real OSGi component. So we dont need to add unused and unnedded dependencies to the buildpath.